analyticsclient/status.py

```diff
@@ -1,20 +1,10 @@
+from analyticsclient._base import BaseEndpoint
 from analyticsclient.exceptions import ClientError
 
 
-class Status(object):
+class Status(BaseEndpoint):
```

Comment:
Really digging the base class refactor a lot after seeing all these repeated `__init__` methods vanish